chain/gssmr/config-bob.toml

```diff
@@ -1,49 +0,0 @@
-[global]
```

Comment:
Why is this file removed?

Reply:
the config is expected to be general purpose.  You can specify accounts via command line.  This is just mirroring what it's like on `development`.